plugins/zeneyer-auth/includes/Auth/class-password-auth.php

```
@@ -334,16 +334,49 @@ private static function validate_password_strength($password) {
+            // If the suffix is numeric, track the max
+            if (ctype_digit($suffix)) {
+                $suffix_int = (int) $suffix;
+                if ($suffix_int > $max_suffix) {
+                    $max_suffix = $suffix_int;
+                }
+            }
         }
         
-        return $username;
+        return $username . ($max_suffix + 1);
     }
```

PHPMD (2.15.0)

[error] The method generate_username is not named in camelCase. (undefined)

(CamelCaseMethodName)